spec/views/tracks/show.html.erb_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "tracks/show", type: :view do
+  before(:each) do
+    @track = assign(:track, Track.create!(
+      :name => "Name",
```

Comment:
Indent the first parameter one step more than `Track.create!(`.<br>Use the new Ruby 1.9 hash syntax.